### PR TITLE
New OpenTelemetry doc about Kafka in Confluent

### PR DESCRIPTION
**Pull request by nr-jbiggley**

### CODE REVIEW

**Comment by LizBaker** [comment]
kicking off a new build

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx [hunk 1]
```diff
@@ -0,0 +1,365 @@
...
+
+## Step 1: Sign up for New Relic! [#signup]
+
+* If you haven't already done so, sign up for a free [New Relic account](https://newrelic.com/signup).
+* Get the [license key](https://one.newrelic.com/launcher/api-keys-ui.launcher) for the New Relic account to which you want to report data.
+
+## Step 2: Prerequisites [#prerequisites]
+
+* Ensure [Go](https://go.dev/doc/install) is installed before proceeding.
+* Set [GOPATH](https://go.dev/doc/gopath_code) to $GOPATH/bin and added to the PATH variable.
```
Comment (akristen) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx:34]:
```suggestion
* Set [GOPATH](https://go.dev/doc/gopath_code) to `$GOPATH/bin` and added to the PATH variable.
```
Not sure if this needs backticks @rhetoric101

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+  - Confluent Cloud
+metaDescription: You can collect Kafka metrics from Confluent using the OpenTelemetry collector.
+---
+You can collect metrics about your Confluent Cloud-managed Kafka deployment with the OpenTelemetry collector. The collector is a component of OpenTelemetry that collects, processes, and exports telemetry data to New Relic (or any observability back-end).
+
+<Callout variant="tip">
+  If you're looking for help with other collector use cases, see the [newrelic-opentelemetry-examples](https://github.com/newrelic/newrelic-opentelemetry-examples) repository.
+</Callout>
+
+Here are the steps you'll complete to collect Kafka metrics from Confluent:
```
Comment (akristen) [inline src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx:17]: This doc is relatively short, so this real estate might be better used explaining in a general, high level way what processes users can expect (what changes are made to the system and where). If that's too redundant, then I'd just remove this section.

**Review by akristen — APPROVED** [review]

File: src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx (hunk 2 above)
Comment (rhetoric101) [reply src/content/docs/more-integrations/open-source-telemetry-integrations/opentelemetry/collector/collector-configuration-examples/opentelemetry-collector-kafka-confluentcloud.mdx]: @akristen I just removed the mini-TOC and used a simple introductory sentence.

**Review by akristen — APPROVED** [review]

**Pull request merged**